### PR TITLE
feat(components): Update apple icon and Add bank icon [JOB-106774]

### PR DESCRIPTION
**Pull request by youre-jobber**
## Motivations

- The previous Apple icon didn't have any built-in padding and it’s throwing off the balance. 
- We are adding a new Bank icon that will be used on the checkout flow 

### Added

This icon:
- bank

### Changed

This icon:
- apple

## Testing

https://github.com/GetJobber/atlantis/actions/runs/11352845839

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Aquamen opening a new bank account

### CODE REVIEW

File: docs/components/Icon/Icon.stories.mdx [hunk 1]
```diff
@@ -326,6 +326,7 @@ use the default icon color.
 | Icon                   | `Name`   |
 | :--------------------- | :------- |
 | <Icon name="apple" />  | `apple`  |
+| <Icon name="bank" />   | `bank`   |
```
Comment (youre-jobber) [inline docs/components/Icon/Icon.stories.mdx:329]: Let me know if you prefer this icon on the `Payment` section

File: packages/design/src/icons/apple.svg [hunk 2]
```diff
@@ -1,3 +1,3 @@
-<svg fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
```
Comment (youre-jobber) [inline packages/design/src/icons/apple.svg:1]: For some reason, the previous Apple icon was 150x150. Do you think we should keep the same size for the same version?

**Review by youre-jobber — COMMENTED** [review]
Another question: Do I need to change the `CHANGELOG.md` file, or is this something that is updated automatically?

File: docs/components/Icon/Icon.stories.mdx (hunk 1 above)
Comment (chris-at-jobber) [reply docs/components/Icon/Icon.stories.mdx:329]: That would be preferable!

**Comment by MichaelParadis** [comment]
@youre-jobber 
1. The change log will be updated automatically.
2. I think the height and width should be set since it isn't set on any of our other icons. Did you follow the exporting of the SVG guide called in "Contributor Guide: Adding an icon to Atlantis" in our confluence?
3. Since Bank is a new icon I think it should live in the payments section instead of the Legacy section https://job-106774-svg-for-apple-ico.atlantis.pages.dev/?path=/docs/components-images-and-icons-icon--docs#legacy

CC: @chris-at-jobber

**Comment by youre-jobber** [comment]
@MichaelParadis, the designer (David) gave me the icon, so I can double-check with him...

**Review by chris-at-jobber — COMMENTED** [review]
Design approved!

**Comment by MichaelParadis** [comment]
@youre-jobber I chatted with @chris-at-jobber and you should be good. You will need to update some snapshots with `npm run test -- -u`

**Review by MichaelParadis — CHANGES_REQUESTED** [review]
Only thing needed to update some snapshots with `npm run test -- -u`

**Review by MichaelParadis — APPROVED** [review]
LGTM!

**Pull request merged**